### ISSUE TITLE
[Bug] Archer/Bard Iron Jaws Feature spell order

### ISSUE THREAD

**Issue by RosskoCholakov** [MKhayle/XIVComboExpanded#486]
Hello. First let me tell you that I love the addon. Without it, I wouldn't enjoy the game as much.
Now I had doubts about how to list that comment - bug or a feature, but at the end I decided to list it as a Bug.
I am returning player and I... returned... today. Decided to give my ol' Bard a go in the leveling roulette. Before that I set up the addon and everything, including the Iron Jaws feature.

As you may know, we learn Venomous Bite at level 6, and Windbite at level 30. Since I was deleveled below level 30, I couldn't DoT the target, because Windbite is first in the order of the sequence, I presume, and the macro stayed at Windbite (grayed out) since it's not on the target, but at the same time I didn't "know" it yet so I couldn't cast it on the target so it can switch to Venomous Bite. Can we move Venomous Bite as the first one? I know in most cases not DoTing a target is quite irrelevant, but still - feedback. Sorry if I created any inconvinience or missed something completely obvious in the settings.

**Comment by MKhayle** [comment]
@kaedys Looks like the Level Sync check is lacking for the Iron Jaws Feature ; however it works correctly for the Locked Iron Jaws (Accessibility combo) feature. 

Thanks for the report, I'll check that when I'll have more time to go deeper in the code!

**Comment by MKhayle** [comment]
This will be fixed with the next update, please wait until I have time to commit a new version!

**Comment by kaedys** [comment]
Whoopsie >.<
